ftw/copymovepatches/tests/test_catalog_fixes_view.py

```diff
@@ -68,6 +70,16 @@ def break_object(self, obj):
         # Change UID so that brain metadata is incorrect.
         # This used to happen on copy/paste without the patch provided
         # by this package.
-        setattr(obj, config.UUID_ATTR, getUtility(IUUIDGenerator)())
+        self._change_uid(obj)
 
         transaction.commit()
+
+    def _change_uid(self, obj):
+
+        # Change UID for Archetypes:
+        if hasattr(obj, config.UUID_ATTR):
+            setattr(obj, config.UUID_ATTR, getUtility(IUUIDGenerator)())
+
+        # Change UID for Dexterity:
+        if hasattr(obj, ATTRIBUTE_NAME):
```

Comment:
@jone, is this the right attribute for DX content?

Reply:
I think so; if it works, it is 😉